.github/codeql-config.yml

```diff
@@ -0,0 +1,8 @@
+paths-ignore:
+  - "editor/js/libs/**/*.*"
+  - "examples/jsm/libs/**/*.*"
+  - "examples/js/libs/**/*.*"
+  - "examples/jsm/loaders/ifc/**/*.*"
+  - "examples/js/loaders/ifc/**/*.*"
+  - "build/*.*"
+  - "**/3rdparty/**"
```

Comment:
I don't think we need this line? 🤔

Reply:
Without this line, you'll get an [extraordinarily large number of alerts](https://github.com/mrdoob/three.js/pull/25056/checks?check_run_id=9844725516) in the `manual/3rdparty/monaco-editor` subtree. The CodeQL analysis in LGTM automatically excluded minified files ([example](https://github.com/mrdoob/three.js/blob/dev/manual/3rdparty/monaco-editor/min/vs/basic-languages/markdown/markdown.js)), but it doesn't do that by default on GitHub.

But of course it's up to you! If you're interested in seeing these alerts and fixing them, then that's great :rocket: